### PR TITLE
Docs/canary

### PR DESCRIPTION
**Pull request by maciejkorsan**

### CODE REVIEW

File: docs/cheatsheet/web/index.html [hunk 1]
```diff
@@ -458,8 +458,8 @@ <h4>Helper methods</h4>
     <section data-tag="cli" class="c-column">
       <h3>Basics</h3>
       <h4>Install</h4>
-      <pre><code class="lang-bash">npm install --global yarn
-npm install -g syncano-cli</code></pre>
+      <pre><code class="lang-bash">
+npm install -g @syncano/cli</code></pre>
```
Comment (mkucharz) [inline docs/cheatsheet/web/index.html]: Proper way now is to install CLI as a private dependency for your project - so it is not a global install.

File: docs/cookbook/web/common/contributing.md [hunk 2]
```diff
@@ -59,27 +59,25 @@ npm run e2e:tutorial -- src/tests/e2e/tutorials/quickstart.test-e2e.js
 
 #### Prerequisites
 
-  * You have `node` v7.0.0+ installed and `yarn` at v0.17.10+ (as we use it for development)
-  * You are familiar with `npm`/`yarn`
+  * You have `node` v7.0.0+ installed (as we use it for development)
```
Comment (mkucharz) [inline docs/cookbook/web/common/contributing.md]: Take a look into main `package.json` and engines requirements.

File: docs/docs/_coverpage.md [hunk 3]
```diff
@@ -27,9 +27,8 @@
           </div>
         </div>
         <div class="terminal-text">
-          <p>
-            <span class="lite">⬢</span> $ npm install -g yarn<br />
-            <span class="lite">⬢</span> $ npm install -g syncano-cli<br />
+          <p> 
+            <span class="lite">⬢</span> $ npm install -g @syncano/cli<br />
```
Comment (mkucharz) [inline docs/docs/_coverpage.md]: Not a global anymore.

File: docs/docs/_navbar.md [hunk 4]
```diff
@@ -1,9 +1,7 @@
...
-
 - Resources
-  - [Syncano CLI](https://github.com/Syncano/syncano-node-cli)
-  - [Syncano Client library](https://github.com/Syncano/syncano-client-js)
-  - [Syncano Server library](https://github.com/Syncano/syncano-server-js)
+  - [Cheatsheet](https://cheatsheet.syncano.io/#/)
+  - [Cookbook](https://cookbook.syncano.io/#/) 
+
+- Canary version
+  - [v. 0.6](https://syncano.github.io/syncano-node-cli/#/)
```
Comment (mkucharz) [inline docs/docs/_navbar.md]: `syncano-node-cli` is an old CLI `syncano-node` is our current repo.

File: docs/docs/building-sockets/basics.md [hunk 5]
```diff
@@ -62,44 +65,71 @@ syncano-cli list hello-world
...
 ```
     socket synced: 17:10:06 hello-world 3943 ms
 ```
+
+## Syncano Socket Hot Deployment
+
+You can use a hot deployment feature which watches your socket's files and deploys it on change (you need to deploy your socket first in order to use this feature). If no socket name is provided it watches all of your sockets.
+
+```sh
+syncano-cli hot <socketname>
```
Comment (mkucharz) [inline docs/docs/building-sockets/basics.md]: To use local version of CLI we should use `npx syncano-cli <command>`. I want to find the way to have some alias no `npx` will not be necessary but so far it is (to run local executables).

File: docs/docs/server-lib-reference/installation.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ cat <my-project>/syncano/<my-socket-name>/package.json
 }
 ```
 
-If `syncano-server` is not listed there you can use `yarn` to install it (you have to be inside Socket folder):
+If `syncano-server` is not listed there you can use `npm` to install it (you have to be inside Socket folder):
```
Comment (mkucharz) [inline docs/docs/server-lib-reference/installation.md]: `syncano-server` is now `@syncano/core`.

File: docs/cookbook/web/common/contributing.md (hunk 2 above)
Comment (maciejkorsan) [reply docs/cookbook/web/common/contributing.md]: we're removing this section from cookbook, but I'll update  corresponding file in docs

File: docs/docs/_navbar.md (hunk 4 above)
Comment (maciejkorsan) [reply docs/docs/_navbar.md]: it's a link to old docs, not to the current repo

**Comment by mkucharz** [comment]
LGTM

File: docs/docs/getting-started/intro.md [hunk 7]
```diff
@@ -15,12 +15,12 @@ So, starting from the very top of the diagram:
...
-- `Server-side Library` (*syncano-server*) - used inside the Socket's scripts for communication with Core Syncano Services (e.g. build-in database)
+- `Client Library` (*@syncano/lib-js-client*) - library used on the client-side (application, website etc.)
+- `Server-side Library` (*@syncano/lib-js-core*) - used inside the Socket's scripts for communication with Core Syncano Services (e.g. build-in database)
 
 ### The Syncano CLI
 
-Use the Syncano Command Line interface (CLI) to manage the whole build and deploy process. You can get it from npm with `npm install syncano-cli -g` command (it requires Node and Yarn to be installed). It's a powerful terminal tool, so you won't have to leave your working environment. The whole synchronization and deployment process happens automatically and seamlessly.
+Use the Syncano Command Line interface (CLI) to manage the whole build and deploy process. You can get it from npm with `npm install @syncano/cli -g` command (it requires Node to be installed). It's a powerful terminal tool, so you won't have to leave your working environment. The whole synchronization and deployment process happens automatically and seamlessly.
```
Comment (adamwardecki) [inline docs/docs/getting-started/intro.md]: unnecessary `-g` flag

File: docs/docs/getting-started/quickstart.md [hunk 8]
```diff
@@ -26,27 +26,25 @@ cd chat-app/
...
 
 # Sign up and create a project
 Once the Syncano CLI is installed you can sign up for an account. Run the following command in the root directory of your project:
 
 ```sh
-syncano-cli init
+npx syncano-cli init
 ```
 
+> There's shortcut for `syncano-cli` - you can simple type `s`
```
Comment (adamwardecki) [inline docs/docs/getting-started/quickstart.md]: s/simple/simply

File: docs/docs/getting-started/windows.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+# Windows Development Guide
+
+For more comfortable development process  on Microsoft Windows, please install Windows Subsystem for Linux. You can find a detailed guide how to do this here: [docs.microsoft.com/en-us/windows/wsl/install-win10](https://docs.microsoft.com/en-us/windows/wsl/install-win10). 
+
+You will also need a working nodejs enviroment. Please follow this guide: [hackernoon.com/running-nodejs-on-linux-on-windows-88bd12993bae](https://hackernoon.com/running-nodejs-on-linux-on-windows-88bd12993bae).
```
Comment (adamwardecki) [inline docs/docs/getting-started/windows.md]: s/enviroment/environment

File: docs/cookbook/web/common/contributing.md [hunk 10]
```diff
@@ -59,27 +59,25 @@ npm run e2e:tutorial -- src/tests/e2e/tutorials/quickstart.test-e2e.js
...
 #### Branching flow
 
-We branch our work from `devel`.
-After merging your changes, they (changes) will wait for our `devel` tests to pass and then automatically merged into `beta` branch.
-It will be submitted to npm with `@beta` tag. From there we will merge it into `master` that will bump proper version of `CLI`.
+We branch our work from `master`.
+After merging your changes, they (changes) will wait for our `master` tests to pass and then automatically merged into `canary` branch.
+It will be submitted to npm with `@canary` tag. From there we will merge it into `master` that will bump proper version of `CLI`.
 
 #### Waiting for review
```
Comment (adamwardecki) [inline docs/cookbook/web/common/contributing.md]: The parts below about commiting and the release process are no longer valid

Comment (maciejkorsan) [reply docs/cookbook/web/common/contributing.md]: @adamwardecki any suggestions on this? how does it look now?

Comment (adamwardecki) [reply docs/cookbook/web/common/contributing.md]: - The whole section about commiting should be removed
- we don't use commitizen, although commits should have a conventional commit style https://conventionalcommits.org/
- we don't use semantic release. seems that lerna handles the release process (It's @mkucharz who configured the set up, so he'll be able to tell more about it)

I see this section is in two places `docs/common` and `cookbook/common`

**Comment by mkucharz** [comment]
@maciejkorsan I think that maybe we should try to close it soon ;)

**Comment by maciejkorsan** [comment]
@mkucharz have a look

**Comment by mkucharz** [comment]
@maciejkorsan let's do it! :)

**Pull request merged**